test/test_images/probe_helper/probe/helper_test.go

```diff
@@ -468,6 +500,22 @@ func TestProbeHelper(t *testing.T) {
 				wantResult: cloudevents.ResultNACK,
 			},
 		},
+	}, {
+		name: "PingSource probe",
+		steps: []eventAndResult{
+			{
+				event:      probeEvent("pingsource-probe", withProbeExtension("period", "200ms")),
+				wantResult: cloudevents.ResultACK,
+			},
+		},
+	}, {
+		name: "PingSource delay exceeds period",
+		steps: []eventAndResult{
+			{
+				event:      probeEvent("pingsource-probe", withProbeExtension("period", "0s")),
+				wantResult: cloudevents.ResultNACK,
+			},
+		},
 	}, {
```

Comment:
You can improve coverage here a bit by adding a test for "missing period extension" and another for "invalid period format", i.e.,
```golang
event:      probeEvent("pingsource-probe"),
wantResult: cloudevents.ResultNACK,
```
and
```golang
event:      probeEvent("pingsource-probe", withProbeExtension("period", "invalid"),
wantResult: cloudevents.ResultNACK,
```

Reply:
Good idea, I've added both